src/main/java/cz/it4i/parallel/runners/HPCSettings.java

```diff
@@ -25,7 +25,7 @@
 	
 	private String authenticationChoice;
 	
-	private transient String password;
```

Comment:
It needs to be transient.

Reply:
I made the password transient in HPCSettings.